src/vm/docs/mod.rs

```diff
@@ -232,6 +232,29 @@ created by this set of bindings is used for evaluating and return the value of `
     example: "(let ((a 2) (b (+ 5 6 7))) (+ a b)) ;; Returns 20"
 };
 
+const FETCH_VAR_API: SpecialAPI = SpecialAPI { 
+    name: "fetch-var",
+    input_type: "VarName",
+    output_type: "A",
```

Comment:
This should be `Optional(A)`

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1022/commits/25d0181f73efef27237cf95a24be26ff42b1af77